### PR TITLE
Nutrition sm 4real

### PR DESCRIPTION
**Pull request by agtravis**
Got calories displaying on small screen.

After figuring out your code @twopcz I was able to figure out where to insert my code to make it work. It was actually pretty straightforward at that point, however I did change one part of your code. On a small screen, you are clicking on the image in order to view the recipe. If you want to see the photo again, you just click it again. When I ran your code on the image click event listener, obviously it would be making the same API request again and again. So I basically put all that code inside an 'IF' statement that said

`if (
        !document.getElementById('calories' + containerNumber + 'SM')
          .textContent
      ) {`

Which essentially says if the calories paragraph already has a text content, then don't allow this part of the code to run again. I realised at that point that since your code is a toggle feature too, then this applies, even if the user is less likely to actually click the button again. The difference for your code on the fullscreen is that you still need some of that code block to function regardless, so I just wrapped the nutrition function call itself in that condition, whereas for my code on the small screen, the whole block is wrapped in the condition.

TL;DR - nutrition works on smallscreen now too

### CODE REVIEW

**Review by justnle — APPROVED** [review]
Nice, having it not call was actually the next thing on my list to do so appreciate you incorporating that into the code @agtravis !
